lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -337,6 +337,10 @@ void ResourceMap::loadDefaultConfiguration()
   // UniVariatePolynomialImplementation parameters //
   setAsUnsignedInteger( "UniVariatePolynomial-SmallDegree", 400 );
 
```

Comment:
Good idea but what are the units ? pourcent ? inch ? Maybe we could add a "unit" suffix to the RM keys

Btw, it should be referenced in docstrings